utils/flare.py

```diff
@@ -88,8 +89,8 @@ def __init__(self, cmdline=False, case_id=None):
     def check_user_rights():
         if Platform.is_unix() and not os.geteuid() == 0:
             log.warning("You are not root, some information won't be collected")
-            choice = raw_input('Are you sure you want to continue [y/N]? ').lower()
-            if choice not in ['yes', 'y']:
+            choice = raw_input('Are you sure you want to continue [y/N]? ')
+            if "".join(choice.lower().split()) not in ['yes', 'y']:
```

Comment:
I chose this version for this :hammer: :
```bash
In [1]: ' y e s'.strip().lower()
Out[1]: 'y e s'

In [2]: ''.join(' y e s'.lower().split())
Out[2]: 'yes'
```

But I agree, it doesn't really make sense, I will change both of them.